components/python/aiohttp/Makefile

```diff
@@ -99,6 +102,7 @@ TEST_REQUIRED_PACKAGES.python += library/python/frozenlist
 TEST_REQUIRED_PACKAGES.python += library/python/gunicorn
 TEST_REQUIRED_PACKAGES.python += library/python/multidict
 TEST_REQUIRED_PACKAGES.python += library/python/mypy
+TEST_REQUIRED_PACKAGES.python += library/python/pkgconfig
 TEST_REQUIRED_PACKAGES.python += library/python/propcache
```

Comment:
Does it make sense to have pkgconfig and propcache as TEST_REQUIRED_PACKAGES if they are needed to build the package anyways?

Reply:
Such duplicates are harmless and comes from the `aiohttps` project itself.  They are detected automatically by our build framework.

It is possible to remove such duplicates (either by patching `aiohttps` or using some generic approach in our `make-rules` directory), but it would be non-trivial work for almost zero profit.